### PR TITLE
Module_Cms MFTF test improvements

### PR DESCRIPTION
**Pull request by nandhini-nagaraj**
### Description (*)

AdminCreateDuplicatedCmsPageTest - Removed the direct action nodes and replaced them with the Action Group for the existing test. Improves the code quality using the best practices.

### Manual testing scenarios (*)

1. AdminCreateDuplicatedCmsPageTest - Removed the direct action nodes and replaced them with the Action Group for the existing test. Improves the code quality using the best practices.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by nandhini-nagaraj** [comment]
@rogyar Kindly review this ticket.

File: app/code/Magento/Cms/Test/Mftf/ActionGroup/SaveAndDuplicateCMSPageWithSplitButtonActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="SaveAndDuplicateCMSPageWithSplitButtonActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Cms/Test/Mftf/ActionGroup/SaveAndDuplicateCMSPageWithSplitButtonActionGroup.xml]: Please, prefix this action group with the area, so we have in result `AdminSaveAndDuplicateCMSPageWithSplitButtonActionGroup`. Thank you

File: app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifyCmsPageSaveSplitButtonActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="VerifyCmsPageSaveSplitButtonActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifyCmsPageSaveSplitButtonActionGroup.xml]: Please, rename this action group to match the follwing name:

```
AssertAdminCmsPageSaveSplitButtonActionGroup
```

Thank you.

**Review by rogyar — DISMISSED** [review]
Hi @nandhini-nagaraj. Thanks a lot for the great improvement. Please, consider making a few minor adjustments described in the comments below.

Thank you!

File: app/code/Magento/Cms/Test/Mftf/ActionGroup/SaveAndDuplicateCMSPageWithSplitButtonActionGroup.xml (hunk 1 above)
Comment (nandhini-nagaraj) [reply app/code/Magento/Cms/Test/Mftf/ActionGroup/SaveAndDuplicateCMSPageWithSplitButtonActionGroup.xml]: @rogyar I have resolved this as per your comments.

File: app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifyCmsPageSaveSplitButtonActionGroup.xml (hunk 2 above)
Comment (nandhini-nagaraj) [reply app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifyCmsPageSaveSplitButtonActionGroup.xml]: @rogyar I have resolved this as per your comment.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-6965](https://jira.corp.magento.com/browse/ENGCOM-6965) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**